lighthouse-core/gather/gatherers/css-usage.js

```diff
@@ -25,7 +25,8 @@ class CSSUsage extends Gatherer {
   beforePass(options) {
     return options.driver.sendCommand('DOM.enable')
       .then(_ => options.driver.sendCommand('CSS.enable'))
-      .then(_ => options.driver.sendCommand('CSS.startRuleUsageTracking'));
+      .then(_ => options.driver.sendCommand('CSS.startRuleUsageTracking'))
+      .catch(_ => this.failure = 'CSS Usage tracking requires Chrome \u2265 56');
```

Comment:
also this marks all errors from `beforePass` as this error. To be fair, the previous code was anticipating no errors :) but probably better to check that it's `'CSS.startRuleUsageTracking' wasn't found` or whatever so that other possible errors aren't muted